File: desafio-02/deyvi-dev/javascript/primos.js
function primos (num) {
  for (let n = 2; n < num; n++) {
    if (num % n === 0) return false
  }
  return true
}
const recebePrimos = (numeros) => {
  for (let contador = 2; contador < numeros; contador++) { if (primos(contador)) console.log(contador) }
}
recebePrimos(1000)

File: desafio-02/lrlucena/scala/primos.scala
// Crivo de Aristóteles
def crivo(a: Stream[Int]): Stream[Int] = a match {
  case p #:: xs => p #:: crivo(xs.filter(_ % p != 0))
}

// Lista de numeros primos
val primos = 2 #:: crivo(Stream.from(3, 2))

// Pegar os menores do que 100000
primos takeWhile(_ < 100000) foreach println
